vxfreeswitch/voice.py

```diff
@@ -293,6 +355,13 @@ def handle_outbound_message(self, message):
 
         client_addr = message['to_addr']
         client = self._clients.get(client_addr)
+
+        if (client is None and message.get('session_event') ==
+                TransportUserMessage.SESSION_NEW):
+            client = yield self.create_voice_client()
+            yield client.make_call(message['to_addr'])
+            self.register_client(client)
```

Comment:
Ah, I see. I'd prefer us to set the address in ``__init__``. That seems like it will require creating a custom factory instance and protocol that knows about the address. That sucks a bit but I don't see a better way right now.